app/platform/license.go

```diff
@@ -24,13 +24,20 @@ import (
 const (
 	LicenseEnv                = "MM_LICENSE"
 	LicenseRenewalURL         = "https://customers.mattermost.com/subscribe/renew"
+	LicenseRenewalURLTest     = "https://portal.test.cloud.mattermost.com/subscribe/renew"
```

Comment:
Wondering if we should use `CWSURL` config setting here as the base url and not have to keep these two separate urls?

Reply:
Yes, I think that is a good idea.

Reply:
changed